### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by BramVDBe**
add a tweet button and email button in header (next to title, alternatively could be in the navbar). 

tweet button from: https://developer.twitter.com/en/docs/twitter-for-websites/tweet-button/overview
email to button from: https://github.com/gschier/html-share-buttons/blob/master/buttons.html (do we have to add the style information in the head section?)

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -59,6 +61,24 @@
...
+	       href="https://twitter.com/intent/tweet"
+	       data-size="large"
+  	       data-text="CAMEL, a database collection (meta)data of experimental evolution studies using microorganisms"
+               data-url="cameldatabase.com"
+               data-hashtags="experimental evolution,database"
+               data-via="BramVDBe,vibbioinfocore">
+	       Tweet
+	    </a>
+		  
+	    <a href="mailto:?subject=<SUBJECT&body=<BODY>" target="_blank" class="btn share-btn email">
```
Comment (MaybeJustJames) [inline public/index.html:76]:
```suggestion
	    <a href="mailto:?subject=<SUBJECT>&body=<BODY>" target="_blank" class="btn share-btn email">
```

File: public/index.html [hunk 2]
```diff
@@ -59,6 +61,24 @@
 	    <h1>CAMEL Database</h1>
 	    <h2>Compendium of Adaptive Microbial Experiments in the Lab</h2>
 	  </div>
+		
+	  <div id="sharemedia" class="pointer col-md-3 col-lg-2">
+	    <a class="twitter-share-button"  
+	       href="https://twitter.com/intent/tweet"
+	       data-size="large"
+  	       data-text="CAMEL, a database collection (meta)data of experimental evolution studies using microorganisms"
+               data-url="cameldatabase.com"
```
Comment (MaybeJustJames) [inline public/index.html:70]: Indentation

**Review by MaybeJustJames — APPROVED** [review]
Hi @BramVDBe  I only just noticed this. Sorry for the delay. Is it still something you want?
If so, consider the changes I suggested and if you're happy I will merge.

**Review by BramVDBe — COMMENTED** [review]
Hi @MaybeJustJames, this is a request from quite a while ago. I think it slipped out of my mind when I realized Thomas was not coming back earlier and thought that Michael could make the changes once he was working on it. But if this should work than this is fine for me!

**Pull request merged**